testing/installtest/checks_windows.go

```diff
@@ -151,9 +155,11 @@ func getAllowedSIDs(secInfo *windows.SECURITY_DESCRIPTOR) ([]*windows.SID, error
 	aceCount := rs.Field(3).Uint()
 	for i := uint64(0); i < aceCount; i++ {
 		ace := &accessAllowedAce{}
-		ret, _, _ := procGetAce.Call(uintptr(unsafe.Pointer(dacl)), uintptr(i), uintptr(unsafe.Pointer(&ace)))
+		ret, _, err := procGetAce.Call(uintptr(unsafe.Pointer(dacl)), uintptr(i), uintptr(unsafe.Pointer(&ace)))
 		if ret == 0 {
-			return nil, fmt.Errorf("while getting ACE: %w", windows.GetLastError())
+			if err = util.FilterErrno(err); err != nil {
```

Comment:
Is this even needed here? Why would it return a 0 error and the last error be set as success?

Reply:
I think the reason we where seeing this before was because of the issue you found in this work. That golang was reading last error when it was not being set.

Reply:
actually you are right the return value is just a bool for this one thus there is not point to check GetLastError(), that golang always returns, with ERROR_SUCCESS. removing it now

Reply:
just removed, please have another pass 🙂